focuspoints.lrdevplugin/Utils.lua

```diff
@@ -46,18 +46,58 @@ function splitText(str, delim)
   return r
 end
 
+function split(str, delim)
```

Comment:
I'll start doc'ing too. and I'll go back and add docs to my old methods as well soon.